+jrclust/+detect/@DetectController/detectOneRecording.m

```diff
@@ -119,48 +119,7 @@
         else
             nSamples = nSamplesLoad;
         end
-
```

Comment:
I'm having trouble making sense of what's changed here except for skipping a load if it doesn't contain any import times. Everything in the diff is shifted around. Can you summarize?

Reply:
That's all it is. If you don't skip the load but there are no times to import, the code was applying its detection algorithm to find spikes in that load, which is obviously not desirable. For efficiency, I moved the code which determines if there are spikes in that load to the very top of the function so it doesn't even both loading the data for that batch unless it needs to. That's why the code doesn't line up anymore.

Reply:
> the code was applying its detection algorithm to find spikes in that load, which is obviously not desirable.

Nice catch!